lms/djangoapps/support/docs/decisions/0001-sso-history-in-support.rst

```diff
@@ -0,0 +1,34 @@
+1. Registering SSO History Model in Support
+================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+SSO History was one of the feature requested by support that provides the 
+historical data of any particular SSO record (based off UserSocialAuth)
```

Comment:
- Apart from support's ask, we can add more details on what benefits it can provide  (tracking id changes, SSO updates over a period of time, etc.). 
- Here, the decision on why the support app was selected over other LMS apps should be elaborated to make it clear to a reader.
- Additionally, we should avoid referencing repos and add a more general word for the tools.